include/deal.II/grid/tria.h

```diff
@@ -2226,26 +2226,12 @@ class Triangulation : public Subscriptor
   /** @} */
 
   /**
-   * Alias for backward compatibility.
+   * @copydoc ::dealii::CellStatus
```

Comment:
Just copy the documentation by hand. It's going to go away after the next release anyway, so I'm not so concerned about the duplication.

Reply:
@bangerth Will do.